TowerForge/TowerForge/GameViewController.swift

```diff
@@ -35,7 +35,7 @@ class GameViewController: UIViewController {
     }
 
     private func setUpGameWorld(scene: GameScene) {
-        self.gameWorld = GameWorld(scene: scene)
+        self.gameWorld = GameWorld(scene: scene, screenSize: self.view.frame)
```

Comment:
I think this is currently fine but I think we are going to implement a pannable screen later on right? Hence, we should define the size in the game world instead.